libs/wingc/src/type_check/symbol_env.rs

```diff
@@ -145,18 +188,32 @@ impl SymbolEnv {
 		}
 	}
 
-	pub fn lookup_nested_str(&self, nested_str: &str, statement_idx: Option<usize>) -> Result<&SymbolKind, TypeError> {
+	pub fn lookup_nested_str(
+		&self,
+		nested_str: &str,
+		ignore_hidden: bool,
+		statement_idx: Option<usize>,
+	) -> Result<&SymbolKind, TypeError> {
 		let nested_vec = nested_str
 			.split('.')
 			.map(|s| Symbol {
 				name: s.to_string(),
 				span: WingSpan::global(),
 			})
 			.collect::<Vec<Symbol>>();
-		self.lookup_nested(&nested_vec.iter().collect::<Vec<&Symbol>>(), statement_idx)
+		self.lookup_nested(
+			&nested_vec.iter().collect::<Vec<&Symbol>>(),
+			ignore_hidden,
+			statement_idx,
+		)
 	}
 
-	pub fn lookup_nested(&self, nested_vec: &[&Symbol], statement_idx: Option<usize>) -> Result<&SymbolKind, TypeError> {
+	pub fn lookup_nested(
+		&self,
+		nested_vec: &[&Symbol],
+		ignore_hidden: bool,
+		statement_idx: Option<usize>,
+	) -> Result<&SymbolKind, TypeError> {
```

Comment:
Maybe some kind of comment like
```suggestion
    /// Pass `ignore_hidden: true` if it's OK to return types that have only been imported implicitly (such as through an inheritance chain), and false otherwise
	pub fn lookup_nested(
		&self,
		nested_vec: &[&Symbol],
		ignore_hidden: bool,
		statement_idx: Option<usize>,
	) -> Result<&SymbolKind, TypeError> {
```